### ISSUE TITLE
RDS Migrations: DynamicFileMigrationProvider attempts to copy migration scripts to separate read-only folder

### ISSUE THREAD

**Issue by mendellerner** [serverless-stack/serverless-stack#1515]
Attempting to run migrations on a non-local stack causes the following error because it tries to copy the migrations to another folder before running.
```
[Error: EROFS: read-only file system, copyfile '/var/task/sst_rds_migration_scripts/migration.2022-03-10.js' -> '/var/task/sst_rds_migration_scripts/migration.2022-03-10.js1646949820654.js'] {
  errno: -30,
  code: 'EROFS',
  syscall: 'copyfile',
  path: '/var/task/sst_rds_migration_scripts/migration.2022-03-10.js',
  dest: '/var/task/sst_rds_migration_scripts/migration.2022-03-10.js1646949820654.js'
}
```

**Comment by mendellerner** [comment]
This bug is fixed with the additional commits in #1511

**Comment by thdxr** [comment]
I actually fixed this in #1508
